routes/products.js

```diff
@@ -0,0 +1,125 @@
+const router = require('express').Router();
+let Product = require('../models/product.model');
+const jwt = require('jsonwebtoken');
+
+router.route('/').get(async (req, res) => {
+  try {
```

Comment:
you can put routers in different file and there use as

import { getProduct } from '../service/product';

router.route('/').get(async(req, res) => getProduct)

const getProduct = async (req, res) => {
  //. logic here
}